src/views/UserManagement.vue

```diff
@@ -418,43 +417,44 @@ export default defineComponent({
         data: this.newUser,
       })
       this.setLoadingDocStatus(false)
-      this.isLoading = false
+      this.isSaveAddUserLoading = false
+      this.resetNewUserInputs()
       this.addShown = false
       this.loadUsers()
     },
 
     async saveRequestEditUser() {
       this.setLoadingDocStatus(true)
-      this.isLoading = true
-      const userId = this.userById.id
-      delete this.userById.id
-      delete this.userById.org_id
-      delete this.userById.org_name
-      delete this.userById.email
+      this.isSaveEditUserLoading = true
+      const userId = this.editUserById.id
+      this.editUserById.id = null
+      this.editUserById.org_id = null
+      this.editUserById.org_name = null
+      this.editUserById.email = null
```

Comment:
We can nullify the entire object at the same time

```suggestion
      this.editUserById = null
```